test/test_session.py

```diff
@@ -1249,6 +1249,7 @@ def test_cluster_time(self):
         # Prevent heartbeats from updating $clusterTime between operations.
         client = rs_or_single_client(event_listeners=[listener],
                                      heartbeatFrequencyMS=999999)
+        self.addCleanup(client.close)
```

Comment:
This should resolve this issue seen in a patch build:
```
 [2020/07/21 22:52:35.405]   Topology <Topology <TopologyDescription id: 5f1771847310f669a600046d, topology_type: ReplicaSetWithPrimary, servers: [<ServerDescription ('localhost', 27017) server_type: RSSecondary, rtt: 0.0011249029994360171>, <ServerDescription ('localhost', 27018) server_type: RSPrimary, rtt: 0.0009582279999449383>, <ServerDescription ('localhost', 27019) server_type: RSArbiter, rtt: 0.0005968250006844755>]>> has THREADS RUNNING: [<PeriodicExecutor(name=pymongo_server_monitor_thread) object at 0x30e7a60>, <PeriodicExecutor(name=pymongo_server_rtt_thread) object at 0x30db0f8>, <PeriodicExecutor(name=pymongo_server_monitor_thread) object at 0x30e7c90>, <PeriodicExecutor(name=pymongo_server_rtt_thread) object at 0x30db478>, <PeriodicExecutor(name=pymongo_server_monitor_thread) object at 0x30dae58>, <PeriodicExecutor(name=pymongo_server_rtt_thread) object at 0x30db168>], created at:   File "setup.py", line 430, in <module>
 [2020/07/21 22:52:35.405]     **extra_opts
 [2020/07/21 22:52:35.405]   File "/opt/python/pypy3.5/site-packages/setuptools/__init__.py", line 161, in setup
 [2020/07/21 22:52:35.405]     return distutils.core.setup(**attrs)
 [2020/07/21 22:52:35.405]   File "/opt/python/pypy3.5/lib-python/3/distutils/core.py", line 148, in setup
 [2020/07/21 22:52:35.405]     dist.run_commands()
 [2020/07/21 22:52:35.405]   File "/opt/python/pypy3.5/lib-python/3/distutils/dist.py", line 955, in run_commands
 [2020/07/21 22:52:35.405]     self.run_command(cmd)
 [2020/07/21 22:52:35.405]   File "/opt/python/pypy3.5/lib-python/3/distutils/dist.py", line 974, in run_command
 [2020/07/21 22:52:35.405]     cmd_obj.run()
 [2020/07/21 22:52:35.405]   File "setup.py", line 130, in run
 [2020/07/21 22:52:35.405]     result = runner.run(suite)
 [2020/07/21 22:52:35.405]   File "/data/mci/6322e5fe30c18cd0207b1725894fa90b/src/test/__init__.py", line 874, in run
 [2020/07/21 22:52:35.405]     result = super(PymongoXMLTestRunner, self).run(test)
 [2020/07/21 22:52:35.405]   File "/opt/python/pypy3.5/site-packages/xmlrunner/runner.py", line 66, in run
 [2020/07/21 22:52:35.405]     test(result)
 [2020/07/21 22:52:35.405]   File "/opt/python/pypy3.5/lib-python/3/unittest/suite.py", line 84, in __call__
 [2020/07/21 22:52:35.405]     return self.run(*args, **kwds)
 [2020/07/21 22:52:35.405]   File "/opt/python/pypy3.5/lib-python/3/unittest/suite.py", line 122, in run
 [2020/07/21 22:52:35.405]     test(result)
 [2020/07/21 22:52:35.405]   File "/opt/python/pypy3.5/lib-python/3/unittest/case.py", line 649, in __call__
 [2020/07/21 22:52:35.405]     return self.run(*args, **kwds)
 [2020/07/21 22:52:35.405]   File "/opt/python/pypy3.5/lib-python/3/unittest/case.py", line 601, in run
 [2020/07/21 22:52:35.405]     testMethod()
 [2020/07/21 22:52:35.405]   File "/data/mci/6322e5fe30c18cd0207b1725894fa90b/src/test/utils.py", line 743, in wrapper
 [2020/07/21 22:52:35.405]     return wrapped(*args, **kwargs)
 [2020/07/21 22:52:35.405]   File "/data/mci/6322e5fe30c18cd0207b1725894fa90b/src/test/test_session.py", line 1251, in test_cluster_time
 [2020/07/21 22:52:35.405]     heartbeatFrequencyMS=999999)
 [2020/07/21 22:52:35.405]   File "/data/mci/6322e5fe30c18cd0207b1725894fa90b/src/test/utils.py", line 500, in rs_or_single_client
 [2020/07/21 22:52:35.405]     return _mongo_client(h, p, **kwargs)
 [2020/07/21 22:52:35.405]   File "/data/mci/6322e5fe30c18cd0207b1725894fa90b/src/test/utils.py", line 461, in _mongo_client
 [2020/07/21 22:52:35.405]     **client_options)
 [2020/07/21 22:52:35.405]   File "/data/mci/6322e5fe30c18cd0207b1725894fa90b/src/pymongo/mongo_client.py", line 726, in __init__
 [2020/07/21 22:52:35.405]     direct_connection=options.direct_connection)
 [2020/07/21 22:52:35.405]   File "/data/mci/6322e5fe30c18cd0207b1725894fa90b/src/pymongo/settings.py", line 72, in __init__
 [2020/07/21 22:52:35.405]     self._stack = ''.join(traceback.format_stack())
```
https://evergreen.mongodb.com/task/mongo_python_driver_tests_python_version_rhel62_test_ssl__platform~rhel62_auth~noauth_ssl~nossl_python_version~pypy3.5_coverage~coverage_test_latest_replica_set_patch_98f8470b5b79049949b5636e0882a3a40c753f70_5f176fa32a60ed3c0bd45dac_20_07_21_22_44_18